src/components-v2/landing/NoVaults.tsx

```diff
@@ -60,9 +60,9 @@ const NoVaults = ({ network }: Props): JSX.Element => {
         <Typography variant="body2" color="textSecondary">
           Switch to another network to view more vaults.
         </Typography>
-        {[Network.Polygon.toString(), Network.BinanceSmartChain.toString()].includes(network) && (
+        {[Network.Fantom.toString()].includes(network) && (
```

Comment:
these enumerations are strings so i don't think you need the explicit conversion